packages/core-utils/src/time.js

```diff
@@ -1,10 +1,60 @@
-import moment from "moment-timezone";
+import {
+  startOfDay,
+  add,
+  format,
+  formatDuration as dateFnsFormatDuration
+} from "date-fns";
+import { utcToZonedTime } from "date-fns-tz";
 
 // special constants for making sure the following date format is always sent to
 // OTP regardless of whatever the user has configured as the display format
 export const OTP_API_DATE_FORMAT = "YYYY-MM-DD";
+// Date-Fns uses a different string format that moment.js
```

Comment:
```suggestion
// Date-Fns uses a different string format than moment.js
```

Reply:
Whoops! Fixed in 6db148be6701f51c7a26b67765a8b15409bb63de